src/client/src/client_main.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// Created by kevin on 6/25/22.
+//
+#include <ctime>
+#include <cstring>
+#include <ros/ros.h>
+#include <client/current_time.h>
+#include <client/login.h>
+#include <iostream>
+
+using namespace std;
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Do not use namespace using-directives.  Use using-declarations instead.  [build/namespaces] [5]